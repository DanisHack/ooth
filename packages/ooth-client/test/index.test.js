import OothClient from '../src'

describe('OothClient', () => {
    test('can construct', () => {
        const client = new OothClient({
            oothUrl: 'http://localhost:3000/auth',
        })
        expect(client).toMatchSnapshot()
    })

    test('can construct standalone', () => {
        const client = new OothClient({
            oothUrl: 'http://localhost:3000/auth',
            standalone: true,
            apiLoginUrl: 'http://localhost:3000/login',
            apiLogoutUrl: 'http://localhost:3000/logout',
        })
        expect(client).toMatchSnapshot()        
    })
})
