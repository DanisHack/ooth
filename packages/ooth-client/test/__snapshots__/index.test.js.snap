// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OothClient can construct 1`] = `
OothClient {
  "oothUrl": "http://localhost:3000/auth",
  "standalone": undefined,
}
`;

exports[`OothClient can construct standalone 1`] = `
OothClient {
  "apiLoginUrl": "http://localhost:3000/login",
  "apiLogoutUrl": "http://localhost:3000/logout",
  "oothUrl": "http://localhost:3000/auth",
  "standalone": true,
}
`;
